tests/test_api.py

```diff
@@ -1346,16 +1345,6 @@ def test_static_page_renders(self):
         assert response.status_code == 200
         assert len(response.data) > 100
 
-    def test_exception_captured_by_sentry(self):
```

Comment:
This test seems relevant, it tests that we are sending errors to sentry. What was the reason to remove it?

Reply:
Well, this tests an endpoint that won't exist anymore. I don't think it's necessary for us to test the Sentry SDK.

Reply:
I don't think that's the goal of the test.
What it is testing is that we do not introduce regressions that would make us skip error collections thus making us blind during deployment. It is not testing the endpoint. The endpoint exists for this test.
You can see it as a sanity test that our monitoring is in place and working. It does not test that the sdk itsefl, it ensures that our dashboard will be reliable. Which is important.

Reply:
Sure. I find it pretty strange to expose an API endpoint for the purpose of testing our Sentry SDK though.

Reply:
yeah, that's another issue, I am sure that same test could be done by sending an invalid query